homeassistant/components/netgear_lte/const.py

```diff
@@ -0,0 +1,22 @@
+"""Constants for the Netgear LTE integration."""
+import logging
+from typing import Final
+
+ATTR_HOST = "host"
+ATTR_SMS_ID = "sms_id"
+ATTR_FROM = "from"
+ATTR_MESSAGE = "message"
+ATTR_FAILOVER = "failover"
+ATTR_AUTOCONNECT = "autoconnect"
+AUTOCONNECT_MODES = ["never", "home", "always"]
+
+CONF_BINARY_SENSOR: Final = "binary_sensor"
+CONF_NOTIFY: Final = "notify"
+CONF_SENSOR: Final = "sensor"
```

Comment:
```suggestion
```
Maybe you can use Platform.BINARY_SENSOR, Platform.NOTIFY and Platform.SENSOR from `homeassistant.const` instead

Reply:
That's true. Furthermore I think the PR looks good, but I am not confident enough to approve it.